src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/DatePicker.cs

```diff
@@ -111,20 +111,7 @@ public  string CalendarIdentifier
 			}
 		}
 		#endif
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
-		[global::Uno.NotImplemented]
-		public  global::System.DateTimeOffset? SelectedDate
-		{
-			get
-			{
-				return (global::System.DateTimeOffset?)this.GetValue(SelectedDateProperty);
-			}
-			set
-			{
-				this.SetValue(SelectedDateProperty, value);
-			}
-		}
-		#endif
+		// Skipping already declared property SelectedDate
```

Comment:
Whats the preferred method here? 
- remove all and add "Skipping already declared property ..."
- or, `#if false || false ...`

Reply:
The 'skipping already declared' comment is added by the generation tool, I don't think we ever add it by hand. (Not that it's wrong to per se, we just don't.)

Changing all the targets to `false` is fine. I _think_ it's ok to just delete the code too but I could be wrong.